src/components/ClientOnly.tsx

```diff
@@ -0,0 +1,19 @@
+"use client"
+
+import { type ReactNode } from "react"
+
+import { useIsClient } from "@/hooks/useIsClient"
+
+type ClientOnlyProps = {
+  children: ReactNode
+  fallback?: ReactNode
```

Comment:
Shouldn't block, but perhaps we rename this to `loading` for consistency with other loading components?